src/mp-units.cpp

```diff
@@ -0,0 +1,69 @@
+module;
+
+// core
+#include <gsl/gsl-lite.hpp>
+#include <mp-units/bits/external/hacks.h>
+#include <array>
+#include <compare>
+#include <concepts>
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+#include <initializer_list>
+#include <iterator>
+#include <limits>
+#include <numbers>
+#include <numeric>
+#include <optional>
+#include <ranges>
+#include <string>
+#include <string_view>
+#include <tuple>
+#include <type_traits>
+#include <utility>
+
+// core-io
+#include <sstream>
+
+// core-fmt
+#include <mp-units/bits/fmt_hacks.h>
```

Comment:
Shouldn't this be in separate submodules? Maybe there are some (e.g. embedded) projects that do not want or can't use this part?